raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -98,6 +98,7 @@ contract MonitoringService is Utils {
         uint256 reward_amount,
         uint256 nonce,
         address monitoring_service_address,
+        bytes memory non_closing_signature,
```

Comment:
Please add
```
/// @param non_closing_signature <description>
```
above.

Reply:
Done. Didn't we have a linter at some point that checked for these docstrings?

Reply:
No, I don't remember.  But if you find one, just bring it in.

Reply:
Ahh, it's in the `compile_contracts`, not in `make lint`. I probably managed to compile the contracts by running the tests instead of calling `compile_contracts`.